assets/common/src/register.py

```diff
@@ -212,7 +214,7 @@ def main():
     }
     json_object = json.dumps(model_info, indent=4)
 
-    registration_file = registration_details_folder/ComponentVariables.REGISTRATION_DETAILS_JSON_FILE
+    registration_file = registration_details_folder / ComponentVariables.REGISTRATION_DETAILS_JSON_FILE
```

Comment:
(nit) Use os.path.join?

Reply:
Since its a `Path` object where `/` is overriden for path join operation, its recommended to use it over `os.path.join`. Both are nearly same on performance aspects

https://docs.python.org/3/library/pathlib.html